packages/turf-transform-scale/index.js

```diff
@@ -56,14 +58,14 @@ module.exports = function (geojson, factor, origin, mutate) {
  * Scale Feature/Geometry
  *
  * @private
- * @param {Feature|Geometry} geojson GeoJSON Feature/Geometry
+ * @param {GeoJSON} geojson GeoJSON Feature/Geometry
  * @param {number} factor of scaling, positive or negative values greater than 0
  * @param {string|Geometry|Feature<Point>|Array<number>} [origin="centroid"] Point from which the scaling will occur (string options: sw/se/nw/ne/center/centroid)
- * @returns {Feature|Geometry} scaled GeoJSON Feature/Geometry
+ * @returns {GeoJSON} scaled GeoJSON object
```

Comment:
If the input is Geometry it outputs a scaled Geometry, we know the outputs/inputs can ONLY be Feature or Geometry.